.github/workflows/images.yaml

```diff
@@ -200,19 +200,16 @@ jobs:
               ${{ env.DOCKER_REPO }}:${{ matrix.release-tag }}
 
           if [[ "${{ matrix.release-tag }}" == "latest" ]]; then
-            # Tag :latest images as :v1.X.Y
             crane cp ghcr.io/${{ github.repository }}@${{ steps.build-and-push.outputs.digest }} \
                 ${{ env.DOCKER_REPO }}:${tag}
           else
-            # Or tag :v1.X.Y-debug and :v1.X.Y-slim
             crane cp ghcr.io/${{ github.repository }}@${{ steps.build-and-push.outputs.digest }} \
                 ${{ env.DOCKER_REPO }}:${tag}-${{ matrix.release-tag }}
           fi
 
-      - if: startsWith(github.ref, 'refs/tags/v')
-        name: Copy Signatures
+      - if: steps.context.outputs.is_release == 'true'
+        name: Sign Docker Hub images
         run: |
-          digest=${{ steps.build-and-push.outputs.digest }}
-          tag="sha256-${digest#sha256:}"
-          crane cp ghcr.io/${{ github.repository }}:${tag} \
-              ${{ env.DOCKER_REPO }}:${tag}
+          cosign sign --yes \
+              --sign-container-identity ghcr.io/${{ github.repository }}@${{ steps.build-and-push.outputs.digest }} \
```

Comment:
where does this ghcr come from? do we need to change the verify instructions in the readme?